GitUI/CommandsDialogs/FormBrowse.cs

```diff
@@ -1833,7 +1904,7 @@ private void FileExplorerToolStripMenuItemClick(object sender, EventArgs e)
             }
             catch (Exception ex)
             {
-                MessageBox.Show(this, ex.Message, Strings.Error, MessageBoxButtons.OK, MessageBoxIcon.Error);
+                MessageBoxes.ShowErrorMessageBox(this, ex.Message);
```

Comment:
It  is not necessary to repeat "MessageBox" in the function name:
```suggestion
                MessageBoxes.ShowError(this, ex.Message);
```